tests/integration/conftest.py

```diff
@@ -16,23 +16,27 @@ def cloud_base_url() -> str:
     return "http://localhost:5000"
 
 
-@pytest.fixture
-def org_id():
-    return os.environ.get("GX_CLOUD_ORGANIZATION_ID")
+@pytest.fixture(scope="module")
+def org_id_env_var() -> str:
```

Comment:
Why are these module scoped?

Reply:
Good question. Since they are used in the module scoped `context` fixture below. It may be a good idea to convert the `context` fixture (and the fixtures it references) to a function scoped fixture to make sure there are no test interdependencies but I haven't yet had a chance to look at all the tests where it is used to confirm.